src/views/ModelPlan/TaskList/Beneficiaries/Frequency/index.tsx

```diff
@@ -48,7 +49,13 @@ const Frequency = () => {
   const { t: h } = useTranslation('draftModelPlan');
   const { modelID } = useParams<{ modelID: string }>();
 
-  const formikRef = useRef<FormikProps<FrequencyFormType>>(null);
+  // Omitting readyForReviewBy and readyForReviewDts from initialValues and getting submitted through Formik
+  type InitialValueType = Omit<
+    FrequencyFormType,
+    'readyForReviewBy' | 'readyForReviewDts'
+  >;
```

Comment:
This is omitting `readyForReviewBy` and `readyForReviewDts` to be submitted in the formik and thus throwing an error with the update mutations. 

This is in each task list